pkg/resource/claim_reconciler.go

```diff
@@ -248,17 +252,19 @@ func WithClaimFinalizer(f ClaimFinalizer) ClaimReconcilerOption {
 // with the supplied manager's runtime.Scheme. The returned ClaimReconciler will
 // apply only the ObjectMetaConfigurator by default; most callers should supply
 // one or more ManagedConfigurators to configure their managed resources.
-func NewClaimReconciler(m manager.Manager, of ClaimKind, with ManagedKind, o ...ClaimReconcilerOption) *ClaimReconciler {
+func NewClaimReconciler(m manager.Manager, of ClaimKind, using ClassKind, with ManagedKind, o ...ClaimReconcilerOption) *ClaimReconciler {
 	nc := func() Claim { return MustCreateObject(schema.GroupVersionKind(of), m.GetScheme()).(Claim) }
+	ns := func() Class { return MustCreateObject(schema.GroupVersionKind(using), m.GetScheme()).(Class) }
```

Comment:
nit: similarly, what does `ns` stand for here? i'm not asking to change it, but some parts of the codebase feel like they've got a bit further with the very terse variable names than I tend to use myself :)

Reply:
Similar to above, just a product of `nc` already being in use for `NewClaim` above. Definitely open to suggestions. Appreciate the attention to details here as accessibility / readability should certainly be prioritized :)